### ISSUE TITLE
PowerShell syntax highlighting ignores backtick escape character

### ISSUE THREAD

**Issue by ygoe** [notepad-plus-plus/notepad-plus-plus#1394]
The following PowerShell code:

```
$str = "Hello `"great`" world."
```

Assigns the string `Hello "great" world.` to the variable `$str`. The backtick is an escape character inside strings. But it is ignored for syntax highlighting in Notepad++, version 6.8.8.

This is especially annoying if you only have one escaped quotation mark in a string (or an odd number) because all following code has the wrong colour, possibly for the rest of the file!

See also: http://ss64.com/ps/syntax-esc.html

**Comment by dail8859** [comment]
I'm not able to run Notepad++ or SciTE currently but looking at the [PowerShell lexer](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/scintilla/lexers/LexPowerShell.cxx) it looks like it doesn't handle any escape characters at all. This would have to be a modification made to Scintilla.

**Comment by dg9ngf** [comment]
Oops. Can somebody please forward this to their issue tracker? For mysterious reasons, my SourceForge accounts never last very long so I consider that ancient platform inaccessible and can't contribute there.

**Comment by MetaChuh** [comment]
we are closing this issue due to multiple newer notepad++ versions since it was filed.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as detailed and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by ygoe** [comment]
This issue seems to depend on #4302 which is not resolved (yet incorrectly marked as closed). Consequently, this issue cannot be closed either. I know GitHub is bad at issue dependency tracking, that's why I've told you so you know.

**Comment by dail8859** [comment]
Fixed in Notepad++ v7.7 since it updated to Scintilla v4.1.4
